src/sentry/relay/config/metric_extraction.py

```diff
@@ -600,6 +589,7 @@ def _log_on_demand_metric_spec(
             "spec_for": spec_for,
             "spec_query_hash": spec_query_hash,
             "spec": spec_dict,
+            "spec_version": spec_version,
```

Comment:
Adding this here is useful in Sentry issues since it can show up in the breadcrumbs.